pages/ag/neuro-rse.md

```diff
@@ -0,0 +1,107 @@
+
+During our meetings, we have brief lightning talks, to share our work with each other, as well as discussions on relevant topics. These topics are determined by group members and may include: how are RSEs managed and incentivized in neuroscience, how to convince scientists of the value of our work (and the time required), what does day-to-day work look like, how to incentivize building / using common resources instead of reinventing the wheel, how to formally run teams and open source collaborations.
+
+We have a Slack channel in the US-RSE space for discussions.
+
+# Questions?
+
+Reach out to the Neuro-RSE coordinators Billy Broderick or David Nicholson on the [US-RSE slack](https://us-rse.org/join/) with questions.
+
+# Active members
+
+Active members attend our regular meetings at least once a quarter (once every 3 meetings).
+
+- [Billy Broderick](wfbroderick.com/), data scientist at [Flatiron Institute's Center For Computational Neuroscience](https://www.simonsfoundation.org/flatiron/center-for-computational-neuroscience/) 
+- David Nicholson
+- [Ryan Ly](https://www.linkedin.com/in/ryanly), scientific data engineer at Lawrence Berkeley National Lab
+- [Adam Tyson](https://adamltyson.com/), Head Research Engineer at the [Sainsbury Wellcome Centre's Neuroinformatics Unit](https://neuroinformatics.dev/)
+  
+*YOUR NAME HERE*
+
+# Resources
+## Neuro-RSE groups
+
+- [NeuroRSE group](https://neurorse.flatironinstitute.org/) at Flatiron Center for Computational Neuroscience
+- [Neuroinformatics Unit](https://neuroinformatics.dev/) at the [Sainsbury Wellcome Centre](https://www.sainsburywellcome.org/web/)
+
+*YOUR GROUP HERE*
+
+## Jobs
+
+- [US-RSE job board](https://us-rse.org/jobs/)
+- [RSE job board ](https://society-rse.org/careers/vacancies/) (international)
+- [Neuromatch job board](https://neuromatch.io/resources/job-board)
+- [SfN job board ](https://neurojobs.sfn.org/jobs/) (skews academic)
+- [Neurorumblr job board ](http://neurorumblr.com/) (skews academic)
+- [world wide neuro job board ](https://www.world-wide.org/jobs/) (skews academic)
+- [neurotechx job board](https://neurotechx.com/find-a-job/)
+- [Academic Data Science Alliance job board ](https://academicdatascience.org/jobs/) (skews academic)
+
+## Other places to find jobs
+
+- [jobRxiv job board](https://jobrxiv.org/) (skews academic)
+- [HigherEdJobs job board](https://www.higheredjobs.com/search/) (skews academic)
+- Institution/company websites
+- The usual job search websites (e.g., Google, Indeed, LinkedIn, Glassdoor)
+- Twitter (seriously)
+
+## Mailing lists
+
+- [comp-neuro mailing list](https://www.cnsorg.org/comp-neuro-mailing-list)
+- [cvnet mailing list](http://nephoscale.ewind.com/mailman/listinfo/cvnet) (skews academic)
+- [visionlist mailing list](http://visionscience.com/mailman/listinfo/visionlist_visionscience.com) (skews academic)
+- [neuroinfo mailing list](https://lists.incf.org/cgi-bin/mailman/listinfo/neuroinfo) (defunct?)
```

Comment:
Looks like the last real post was July 2022
https://lists.incf.org/pipermail/neuroinfo/
https://lists.incf.org/pipermail/neuroinfo/2022-July/thread.html
Suggest we remove

Reply:
Agreed

Reply:
:+1: